install/windows.md

```diff
@@ -752,36 +754,31 @@ Der Test-Befehl lautet:
 texdoc texlive
```

Comment:
Kein XLaunch

Reply:
The XServer is build in XLaunch is not necessary any more.
> Prerequisites
   • You will need to be on Windows 10 Build 19044+ or Windows 11 to access this feature.

from: https://learn.microsoft.com/en-us/windows/wsl/tutorials/gui-apps

Reply:
Looking at https://learn.microsoft.com/en-us/windows/release-health/release-information, it seems all windows 10 versions older than build 19044 are end of life.